python/cugraph/cugraph/datasets/dataset.py

```diff
@@ -183,7 +186,15 @@ def get_edgelist(self, download=False):
             header = None
             if isinstance(self.metadata["header"], int):
                 header = self.metadata["header"]
-            self._edgelist = cudf.read_csv(
+            if create_using is None:
+                reader = cudf
+            elif str(type(create_using)) != "<class 'module'>":
+                raise RuntimeError("create_using must be a module.")
+            elif create_using.__name__ == "cudf" or "pandas":
```

Comment:
if the suggested changes to `create_using` are not used, then this bug needs to be fixed.  This will always evaluate to True, but I think you meant this?
```suggestion
            elif create_using.__name__ in {"cudf", "pandas"}:
```